src/pages/0/0.ts

```diff
@@ -0,0 +1,9 @@
+import { allPages } from '../../utils/content';
+
+// https://github.com/withastro/docs/blob/main/src/pages/0/0.ts
+// https://www.github.com/withastro/docs/pull/4266/commits/030073f32d6dfe586c6e1da8d48d6b5485541ba2
+
```

Comment:
If we can't find an issue, can we open one to point to so we know when this hack may be replaced by an improved implementation.

Reply:
https://www.github.com/withastro/astro/issues/8972